.swiftlint.yml

```diff
@@ -6,11 +6,14 @@ excluded:
   - DerivedData
 
 disabled_rules:
+  - function_parameter_count
+  - nesting
   - type_name
   - identifier_name
   - line_length
   - multiple_closures_with_trailing_closure
   - todo
+  - indentation_width
```

Comment:
Enabling this rule doesn't allow us to have multiline function calls like this one:

![Screenshot 2023-02-03 at 15 00 29](https://user-images.githubusercontent.com/36865532/216634897-e7d9361d-6a51-4e95-a77b-9c62b72e7189.png)

Reply:
Perhaps we should use [`swiftlint --fix`](https://github.com/realm/SwiftLint#auto-correct) on this file to propose a solution that is acceptable to the tool.  If that solution is also acceptable to project maintainers then we should go that way instead of disabling this rule.  If all that succeeds then the code format will align with other swiftlinted projects.

Reply:
This rule has no autocorrection. Warnings appear on completely valid pieces of code.

-----------

Update: found the reason for this warning. We should not place the first argument on the same line as the function name. The code should be instead:


<img width="801" alt="Screenshot 2023-02-05 at 15 33 32" src="https://user-images.githubusercontent.com/36865532/216822160-deab21e2-560e-4ab2-97d1-191285102a5d.png">

Not an unreasonable variation of indentation and style.

But it also applies to `guard` statements (and likely not only). All conditions, if there are multiple, should take a separate line each and must not be on the same line as `guard` keyword:

<img width="827" alt="Screenshot 2023-02-05 at 15 31 20" src="https://user-images.githubusercontent.com/36865532/216822236-836280b5-e3c8-4337-87b5-4c5fe6a7b552.png">

This variation is also allowed by the rule, but it's less readable and not pleasant to look at at all: 

<img width="747" alt="Screenshot 2023-02-05 at 15 36 13" src="https://user-images.githubusercontent.com/36865532/216822350-49e9fb5a-920d-4710-88af-a71ef1f63734.png">

Based on these findings I'll re-enable this rule per @cclauss suggestion. 
@yaroslavyaroslav @janndriessen Please be aware that the examples above are now the style for multiline function calls and guard multiline statements.

Reply:
This rule is all about indentation and is compatible with almost all code in web3swift. 
It doesn't force us to write `guard` statements in multiline always. Here is one of the acceptable examples:
<img width="480" alt="Screenshot 2023-02-05 at 15 41 53" src="https://user-images.githubusercontent.com/36865532/216822750-77fc685f-16e9-43ac-a80d-254327cbbbbc.png">